[14-47-58] [ENTRY]: [Nathan]
[14-47-59] [SYSTEM_IO]: System data pared in IOUtil's static block
[14-47-59] [SYSTEM_IO]: Suggestions pared in IOUtil's static block
[14-47-59] [SYSTEM_IO]: Exit conditions pared in IOUtil's static block
[14-47-59] [SYSTEM_IO]: DebugHashes pared in IOUtil's static block
[14-48-00] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.updateOldJson(UserUtil.java:774)
From line: 774
Trace: java.lang.NullPointerException
	at cyder.utilities.UserUtil.updateOldJson(UserUtil.java:774)
	at cyder.utilities.IOUtil.fixUsers(IOUtil.java:386)
	at cyder.genesis.Cyder.main(Cyder.java:93)
[14-48-04] [CORRUPTION]: userdir saved to: [Resulting Popup]
Unfortunately a user's data file was corrupted and had to be deleted. The following files still exists and are associated with the user at the following path:<br/>Cyder/dynamic/5d5f838e-2cfc-3a60-a36b-26f98ef35cec<br/>Files:<br/>Default
[14-51-15] [EOL]: Log completed, Cyder was force closed by an external entity: exit code: -200 [External Stop], exceptions thrown: 1